0.4.0

igorzi opened the pull request.

lucacasonato reviewed (commented):
Also need to bump time machine

igorzi commented:
> Also need to bump time machine

I saw that `timemachine` and `remote` didn't have any changes since last release, so I left them out. Should I still bump?

lucacasonato reviewed (approved):
LGTM
